javascript-functions-practice/get-character-at-index.js

```diff
@@ -0,0 +1,4 @@
+/* exported getCharacterAtIndex */
+function getCharacterAtIndex(string, index) {
+  return string.charAt(index);
```

Comment:
This is totally okay, but you can also just access the index using brackets just like an array! So you can also just do `return string[index]`. Both ways are totally fine, I just wanted to show you another way!